src/common/constants.ts

```diff
@@ -78,5 +84,16 @@ export const setColumnToBackendTerms: any = (listName: string, columnName: strin
     backendTerms.DESCRIPTION = 'label';
   }
 
-  return backendTerms[columnName];
+  let lastSort = null;
```

Comment:
@mdshamoon We should only define constants here. We should move this logic to List.tsx and define a new method to handle this.